.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+	env: {
+		node: true,
+		browser: true
+	},
+	extends: [
+		'eslint:recommended',
+		'plugin:vue/vue3-recommended',
+	],
+	rules: {
+		'vue/html-indent': ['error', 'tab'],
+		'vue/html-quotes': ['error', 'double'],
+		'vue/max-attributes-per-line': 'off',
```

Comment:
```suggestion
		'vue/max-attributes-per-line': 4
```

I thought you said you wanted these to be max 4